waiters_app/src/Models/Order.ts

```diff
@@ -0,0 +1,16 @@
+import {Location, OrderIdo, OrderStatus} from '../ido';
+
+export default class Order {
+	public readonly id: string;
+	// public readonly guestID: string;
+	// public readonly items: Record<string, number>;
+	public readonly items: string[];
```

Comment:
uncomment guestID

items are 
readonly Map<String,number> 
according to class diagram